lib/membrane_http_adaptive_stream/stream_sink_bin.ex

```diff
@@ -0,0 +1,146 @@
+defmodule Membrane.HTTPAdaptiveStream.StreamSinkBin do
+  @moduledoc """
+  This bin takes a parsed video stream and dumps an HLS playlist produced from it to storage.
+
+  The bin payloads and muxes the incoming stream to the CMAF format and dumps it to an HLS playlist.
+
+  # Input streams
+  ! can this bin handle multiple streams??
+  The bin expects parsed H264 or AAC video streams to be connected via `:input` pads.
+  The type of video stream has to be specified via the `:encoding` option.
+
+  # Output
+  Specify one of `Membrane.HTTPAdaptiveStream.Storages` as `:storage` to configure the sink.
+
+
+  #! generalnie to nie wiem jak pisać dokumentację
```

Comment:
Don't use Polish anywhere. It is supposed to be OSS and English is a must.

Reply:
And if you don't know how to write documentation just browse some other membrane packages and get inspired.